RELEASING.md

```diff
@@ -120,3 +126,19 @@ gem.
    [`google-cloud/google-cloud.gemspec`](https://github.com/googleapis/google-cloud-ruby/blob/master/google-cloud/google-cloud.gemspec).
 1. Add the gem to
    [`gcloud/Gemfile`](https://github.com/googleapis/google-cloud-ruby/blob/master/gcloud/Gemfile).
+
+## Checking the status of kokoro builds
+
+1. On the [commits](https://github.com/googleapis/google-cloud-ruby/commits/master) page, find the commit you are checking for.
+
+1. To the right of your commit, there should be either a red x, or a green checkmark. Click it.
```

Comment:
Why click the green checkmark? If it is reliable proof of success, shouldn't we exit the steps here?